src/modules/configurator/state/configState.ts

```diff
@@ -18,14 +18,24 @@ export const userConfigurationsState = atom<UserCarConfiguration[]>({
 })
 
 export const configurationPriceState = selector({
-  key: "configurator.configurationPrice",
+  key: "configurator.configurationPriceState",
   get: ({get}) => {
-    const configuration = get(userConfigurationState);
+    const userConfiguration = get(userConfigurationState);
     
-    if (!configuration) return;
-    const price = configuration.totalPrice + configuration.color.price + configuration.wheels.price;
-    const decimalPrice = price.toFixed(2).replace(decimalRegexp, ",");
+    if (!userConfiguration) return;
+    const price = userConfiguration.modelPrice + userConfiguration.color.price + userConfiguration.wheels.price + userConfiguration.interior_variants.price;
```

Comment:
When you add the numbers like this, when you have decimal numbers there could be an error with many decimals. To make is safer round it to 2 decimals

Reply:
Set toFixed(2) for each and wrapped in parseFloat.